doma-spring-boot-samples/doma-spring-boot-sample-testcontainers/src/test/java/org/seasar/doma/boot/sample/ApplicationTest.java

```diff
@@ -13,59 +13,72 @@
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpMethod;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
 import org.springframework.web.util.UriComponentsBuilder;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
-@ContextConfiguration(classes = TestcontainersConfiguration.class)
-@ActiveProfiles("test")
+@Testcontainers
 class ApplicationTest {
-    @Autowired
-    private TestRestTemplate restTemplate;
-    private final ParameterizedTypeReference<List<Message>> typedReference = new ParameterizedTypeReference<List<Message>>() {
-    };
-    @LocalServerPort
-    private int port;
+	@Autowired
+	private TestRestTemplate restTemplate;
+	private final ParameterizedTypeReference<List<Message>> typedReference = new ParameterizedTypeReference<List<Message>>() {
+	};
+	@LocalServerPort
+	private int port;
+	
+	@Container
+	static PostgreSQLContainer<?> postgres = new PostgreSQLContainer<>("postgres:15");
+	
+	@DynamicPropertySource
```

Comment:
`@DynamicPropertySource` is no longer needed as JdbcConnectionDetails is provided via Testconatiners support.